demosite/setup.py

```diff
@@ -23,7 +23,7 @@
     zip_safe=False,
     install_requires=[
         'django-oscar-api',
-        'django>=1.8, <2.0',
+        'django>=1.11, <2.1',
```

Comment:
can you not better loose the <2.1 ? There is no reason to exclude versions that have not been tested to be incompatible!

Reply:
This is for the demosite to make sure this works for demonstration purposes. So it's not setup.py of `oscarapi` itself. Still care that I change it? 😄

Reply:
nah